### PR TITLE
[5.6] Add hashing configuration

### PR DESCRIPTION
**Pull request by huguesjoyal**
This is relation for the PR [5.6] Add hashing configuration #23573 on the laravel/framework

### CODE REVIEW

File: config/hashing.php [hunk 1]
```diff
@@ -17,4 +17,43 @@
...
+    | bcrypt options
+    |--------------------------------------------------------------------------
+    |
+    | We could define the number of rounds the bcrypt algo will be using.
+    |
+    | The two digit cost parameter is the base-2 logarithm of the iteration
+    | count for the underlying Blowfish-based hashing algorithmeter and must
+    | be in range 04-31, values outside this range will cause crypt() to fail
+    |
+    | Default: 10
```
[the remark this answers is not in the document]
Comment (huguesjoyal) [reply config/hashing.php]: I know it should be 12, but in the laravel/framework, Hashing/BcryptHasher.php the protected rounds are currently set to 10.

[the remark this answers is not in the document]
Comment (huguesjoyal) [reply config/hashing.php]: Sorry, but it's supposed to be 10 (http://php.net/manual/en/function.password-hash.php)

I'm sending back the newline request

[the remark this answers is not in the document]
Comment (GrahamCampbell) [reply config/hashing.php]: What's not what I meant. The default is not an integer, it's an array.

[the remark this answers is not in the document]
Comment (huguesjoyal) [reply config/hashing.php]: So instead of "Default: 10" you would like to have "rounds: 10", is that it?

File: config/hashing.php [hunk 2]
```diff
@@ -17,4 +17,47 @@
...
+    |--------------------------------------------------------------------------
+    | argon options
+    |--------------------------------------------------------------------------
+    |
+    | These settings could be adjusted depending on your hardware.
+    |
+    | time:     Maximum amount of time it may take to compute the Argon2 hash.
+    |           (default: 2)
+    |
+    | memory:   Maximum memory (in bytes) that may be used to compute the Argon2
```
Comment (GrahamCampbell) [inline config/hashing.php]: This line is too long. :)

Comment (huguesjoyal) [reply config/hashing.php]: Lol, seriously, I'm sending it back right now... what is the max line length? is there any guideline?

File: config/hashing.php [hunk 3]
```diff
@@ -17,4 +17,47 @@
 
     'driver' => 'bcrypt',
 
+    /*
+    |--------------------------------------------------------------------------
+    | bcrypt options
+    |--------------------------------------------------------------------------
+    |
+    | We could define the number of rounds the bcrypt algo will be using.
```
Comment (m1guelpf) [inline config/hashing.php]: We could? I feel like this should be more like "Here you can"

File: config/hashing.php [hunk 4]
```diff
@@ -17,4 +17,47 @@
...
+    'bcrypt' => [
+        'rounds' => 10,
+    ],
+
+    /*
+    |--------------------------------------------------------------------------
+    | argon options
+    |--------------------------------------------------------------------------
+    |
+    | These settings could be adjusted depending on your hardware.
```
Comment (m1guelpf) [inline config/hashing.php]: could?

**Pull request merged**